src/NzbDrone.Api/REST/RestModule.cs

```diff
@@ -214,7 +214,10 @@ protected TResource ReadResourceFromRequest(bool skipValidate = false)
         private PagingResource<TResource> ReadPagingResourceFromRequest()
         {
             int pageSize;
-            int.TryParse(Request.Query.PageSize.ToString(), out pageSize);
+            if (!int.TryParse(Request.Query.PageSize.ToString(), out pageSize))
+            {
+                pageSize = 100000;
```

Comment:
Why are you setting pageSize to 100,000 here? Why not do it 2 lines below where `if(pageSize == 0) pageSize = 10`?